pkg/diskmaker/controllers/lv/diskmaker.go

```diff
@@ -0,0 +1,515 @@
+package lv
+
+import (
```

Comment:
I would name this file `reconciler.go` since this file encapsulates the new reconciler.

Reply:
+1

Reply:
Done